e3/vcs/git.py

```diff
@@ -126,7 +126,12 @@ def init(self, url=None, remote='origin'):
         :raise: GitError
         """
         e3.fs.mkdir(self.working_tree)
-        self.git_cmd(['init'])
+        try:
+            result = self.git_cmd(['init'], output=PIPE)
+        except GitError:
+            logger.error('git init output:\n%s', result.out)
+            raise
```

Comment:
you should not need that given that exceptions and logs are handled by git_cmd

Reply:
done